appintro/src/main/res/layout-land/appintro_fragment_intro.xml

```diff
@@ -13,7 +13,6 @@
         android:id="@+id/title"
         style="@style/AppIntroDefaultHeading"
         android:layout_width="0dp"
-        android:layout_height="wrap_content"
```

Comment:
Why have you removed this?

Reply:
Sorry, I noticed this immediately after the push.

Reply:
Sorry, I noticed this change only after the push. I was trying to undo it.

Reply:
I fixed it